### DISCUSSION TITLE
New Source: Qonto

### DISCUSSION THREAD

**Discussion by leilaballouard** [airbytehq/airbyte#36093] [New Connector Request]
## Tell us about the new connector you’d like to have
Source connector to retrieve data from [Qonto API](https://api-doc.qonto.com/docs/business-api/83e3d6c0404f3-getting-started) and three first streams : 
- [labels](https://api-doc.qonto.com/docs/business-api/c7249f7ee667c-list-labels)
- [memberships](https://api-doc.qonto.com/docs/business-api/bbda8ecfe01f0-list-memberships)
- [transactions](https://api-doc.qonto.com/docs/business-api/2c89e53f7f645-list-transactions)

## Describe the context around this new connector
This new connector would enable to retrieve different data linked to a Qonto account : "[Qonto](https://qonto.com/en) is the all-in-one business account for SMEs and freelancers with a simplified accounting flow" 

Connection could be done using full-refresh or incremental sync, as there is date fields that can be used for this purpose. 

Qonto API is a paid service, but test can be made using the sandbox url : https://thirdparty-sandbox.staging.qonto.co/v2/internal_transfers

## Are you willing to submit a PR?
Yes, it's in progress ! We are already using a custom connector on our side for this use case.

**Comment by leilaballouard** [comment]
Hello Airbyte team ! I'm currently working on it on my side, can it be assign to me ?

**Comment by sajarin** [comment]
Hey @leilaballouard, I assigned the issue to you. Happy to review your PR when it's ready!

**Comment by leilaballouard** [comment]
Hello , I only had time to make a draft PR : https://github.com/airbytehq/airbyte/pull/17452 , I will continue this after my return from holidays (end of october) !

**Comment by sajarin** [comment]
Thanks @leilaballouard, feel free to ping me when you get around to finishing the PR. Enjoy your holiday :)

**Comment by leilaballouard** [comment]
Hey @sajarin , I'm back from holiday ! 
I reopened the PR and it's ready for review.

[a comment by an automated account is not shown]
